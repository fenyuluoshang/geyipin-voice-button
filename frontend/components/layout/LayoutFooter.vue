<script setup lang="ts"></script>

<template>
  <el-footer class="w-full min-h-[60px] bg-[--bg-color] fixed z-[2] bottom-0">
    <div class="flex justify-between h-full pt-[10px] pb-[10px]">
      <div>Â©2024 Fenyu</div>
    </div>
  </el-footer>
</template>
