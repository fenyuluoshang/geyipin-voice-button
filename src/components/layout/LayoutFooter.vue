<script setup lang="ts"></script>

<template>
  <el-footer class="w-full  min-h-[60px] bg-white bg-opacity-60 fixed z-[2] bottom-0">
    <div class="flex justify-between h-full pt-[10px] pb-[10px]">
      <div>©2024 Fenyu</div>
      <div class="flex flex-col items-end text-[--text-color-regular]">
        <p>
          <a href="https://github.com/fenyuluoshang/geyipin-voice-button" class=" hover:text-[--text-color-primary]" target="_blank">
            <github-icon class="w-[20px] inline-block"></github-icon> geyipin-voice-button
          </a>
        </p>
        <p>本站为爱好者作品，与up主没有关联</p>
      </div>
    </div>
  </el-footer>
</template>
