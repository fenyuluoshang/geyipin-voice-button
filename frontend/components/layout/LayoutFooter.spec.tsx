import { describe, expect, it } from 'vitest'

import { mount } from '@vue/test-utils'
import LayoutFooter from './LayoutFooter.vue'

describe('components/layout/LayoutFooter.vue', () => {
  it('render', ()=>{
    const wrapper = mount(LayoutFooter)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
