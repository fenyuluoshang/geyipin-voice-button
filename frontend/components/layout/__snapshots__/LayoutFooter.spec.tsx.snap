// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/layout/LayoutFooter.vue > render 1`] = `
"<footer class="el-footer w-full min-h-[60px] bg-[--bg-color] fixed z-[2] bottom-0">
  <div class="flex justify-between h-full pt-[10px] pb-[10px]">
    <div>Â©2024 Fenyu</div>
  </div>
</footer>"
`;
